### ISSUE TITLE
Wheel not working so good

### ISSUE THREAD

**Issue by Slion** [minhe7735/GearVR-Controller-WIndows#4]
I've increased the sensitivity but it ain't working so good, often scrolls in the wrong direction too.

**Comment by minhe7735** [comment]
Yeah, if sensitivity gets too high it won't work properly unless you move your fingers slowly as well. I'm not sure if that is hardware limitation or errors in my calculation. Would have to test it out. 

As for scrolling in the wrong direction, I haven't had that problem but you do have to move in a circular motion like those scroll wheels on iPods.

**Comment by Slion** [comment]
> As for scrolling in the wrong direction, I haven't had that problem but you do have to move in a circular motion like those scroll wheels on iPods.

Ho I see, that's why I could not get it to work… 😂
